data/incipias/incipias outfits.txt

```diff
@@ -0,0 +1,297 @@
+
+outfit "Metallic Hydrogen Cell"
+	category "Power"
+	thumbnail "outfit/metallic hydrogen cell"
+	mass 20
+	"outfit space" -20
+	"energy capacity" 500
+	description "This form of energy storage was discovered early by the Incipias because of the great amount of metallic hydrogen on their homeworld. Because Incipias ships mostly access their energy at the time it is produced, only a small amount of storage is needed. Most of their ships have only one cell installed as a consequence."
+
+# Engines:
+outfit "MH Blue-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh blue-class engine"
+	"mass" 27
+	"outfit space" -27
+	"engine capacity" -27
+	"flare sprite" "effect/incipias flare/metallic hydrogen small"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 16
+	"thrusting energy" 2
+	"thrusting heat" 2.5
+	description "In Incipian culture, blue is the weakest color. An engine labeled with blue does not bode well for the speed of any ship using it."
+
+outfit "MH Blue-Class Steering"
+	category "Engines"
+	thumbnail "outfit/mh blue-class steering"
+	"mass" 24
+	"outfit space" -24
+	"engine capacity" -24
+	"steering flare sound" "plasma huge"
+	"turn" 332
+	"turning energy" 1.
+	"turning heat" 3
+	description "This steering barely provides the minimum force needed to turn a ship, but it still used often because of how cheap it is."
+
+outfit "MH Green-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh green-class engine"
+	"mass" 32
+	"outfit space" -32
+	"engine capacity" -32
+	"flare sprite" "effect/incipias flare/metallic hydrogen medium"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 20
+	"thrusting energy" 2
+	"thrusting heat" 2.5
+	description "Green is a color of balance for the Incipias. This engine is balanced between taking too much space and moving too slow."
+
+outfit "MH Green-Class Steering"
+	category "Engines"
+	thumbnail "outfit/mh green-class steering"
+	"mass" 28
+	"outfit space" -28
+	"engine capacity" -28
+	"steering flare sound" "plasma huge"
+	"turn" 412
+	"turning energy" 1.1
+	"turning heat" 3.1
+	description "The Incipias seem to have something against turning their ships. Even though this steering is supposed to be an upgrade from the Blue-Class steering, it is barely an improvement."
+
+outfit "MH Red-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh red-class engine"
+	"mass" 41
+	"outfit space" -41
+	"engine capacity" -41
+	"flare sprite" "effect/incipias flare/metallic hydrogen large"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 32
+	"thrusting energy" 5
+	"thrusting heat" 10
+	description "The Incipias burn metallic hydrogen to move around in space. As a race that has easy access to metallic hydrogen on their home planet this allows for very efficient engines."
```

Comment:
Wouldn't easy access to metallic hydrogen encourage less efficient engines, as they can waste fuel with little to no consequence?

Reply:
The point is that metallic hydrogen is a very efficient fuel type